longTrip.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+import math
+
+class Trip:
+    """
+    トリップ情報を保持するクラス
+    
+    Attributes
+    ----------
+    userId: int
+        ユーザID
+    points: dict
+        キーに座標のタイムスタンプ、値にXY座標
+    
+    """
+    
+    userId = -1
+    points = {}
+    
+    def __init__(self, userId):
+        """
+        初期化
+        
+        Attributes
+        ----------
+        userId: int
+            ユーザID
+        """
+        
+        self.userId = userId
+        self.points = {}
+    
+    def addPoint(self, timeStamp, pointX, pointY):
+        """
+        座標の追加
+        
+        Attributes
+        ----------
+        timeStamp: int
+            UNIXタイムでの時刻
+        pointX: int
+            X座標
+        pointY: int
+            Y座標
+        """
+        self.points[timeStamp] = (pointX, pointY)
```

Comment:
座標データは、タプルではなく、座標クラスを作成して、管理すると良いと思います。クラスにすることで、低凝集にできますし、Validationも簡単に実装できると思います。


```suggestion
    class coordinate:
        def __init__(self, x, y):
            self.x = x
            self.y = y
```